Units/parser-javascript.r/bug1950327.js.d/expected.tags

```diff
@@ -1,13 +1,13 @@
-*	input.js	/^		container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}		$/;"	p	class:container.dirtyTab
+*	input.js	/^		container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}		$/;"	p	property:container.dirtyTab
 Different	input.js	/^Different.prototype = {$/;"	c
 TabChrome	input.js	/^TabChrome.prototype = {$/;"	c
+container.dirtyTab	input.js	/^		container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}		$/;"	p
```

Comment:
This is just another two-tokens tag.
I think this should be:
```
dirTab	input.js	/^		container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}		$/;"	p	variable:container
container	input.js	/^		container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}		$/;"	v	roles:chainElt
```

What the scope for `cotainer` is is another issue.